src/Libraries/RevitNodes/Elements/InternalUtilities/ElementWrapper.cs

```diff
@@ -167,6 +167,10 @@ public static Element Wrap(Autodesk.Revit.DB.ViewPlan view, bool isRevitOwned)
             {
                 return FloorPlanView.FromExisting(view, isRevitOwned);
             }
+            else if (view.ViewType == ViewType.EngineeringPlan)
```

Comment:
Please switch this series of `else if` out of a `case` statement. Once I get a second `else if`, I usually do this as a matter of style.